script.js

```diff
@@ -0,0 +1,134 @@
+            {text: "Italy", correct: false},
+            {text: "France", correct: false},
+        ]
+    },
+    {
+        question: "Which is largest continent in the world?",
+        answers: [
+            {text: "Asia", correct: true},
+            {text: "Africa", correct: false},
+            {text: "Australia", correct: false},
+            {text: "Europe", correct: false},
+            ]
+    },
+    {
+        question: "Do I love my handsome boy",
+        answers: [
+            {text: "Yes", correct: true},
+            {text: "No", correct: false},
+        ]
+    },
+    {
+        question: "Which is the longest river in the world?",
+        answers: [
+            {text: "Congo", correct: false},
+            {text: "Yellow", correct: false},
+            {text: "Nile", correct: true},
+            {text: "Niger", correct: false},
+            ]
+    }
+];
+const questionElement = document.getElementById("question");
+const answerButtons = document.getElementById("answer-buttons");
+const NextButton = document.getElementById("next-btn");
+
+let currentQuestionIndex = 0;
+let score = 0;
+
+function startQuiz(){
+    currentQuestionIndex = 0;
+    score = 0;
+    NextButton.innerHTML = "Next";
+    showQuestion();
+}
+
+function showQuestion(){
+    resetState();
+    let currentQuestion = questions[currentQuestionIndex];
+    let questionNo = currentQuestionIndex + 1;
+    questionElement.innerHTML = questionNo + ". " + currentQuestion.question;
+
+    currentQuestion.answers.forEach(answer => {
+        const button = document.createElement("button"); // <button></button>
+        button.innerHTML = answer.text; // <button>{answer.text}</button>, np. <button>Shark</button>
+        button.classList.add("btn"); // <button class="btn">{answer.text}</button>, np. <button class="btn">Shark</button>
+        answerButtons.appendChild(button);
+
+        button.dataset.correct = answer.correct;  // <button class="btn" data-correct="true">{answer.text}</button>, np. <button class="btn" data-correct="true">Shark</button>
+
+        button.addEventListener("click", selectAnswer); // <button class="btn" data-correct="true" onclick="selectAnswer">{answer.text}</button>, np. <button class="btn" data-correct="true" onclick="selectAnswer">Shark</button>
+    });
+}
+
+function resetState() {
+    NextButton.style.display = "none";
+    while(answerButtons.firstChild){
+        answerButtons.removeChild(answerButtons.firstChild);
+    }
+}
+
+function selectAnswer(e){
+    const selectedBtn = e.target;
+    const isCorrect = selectedBtn.dataset.correct === "true";
+    if(isCorrect){
+        selectedBtn.classList.add("correct");
+        score++;
+    }else{
+        selectedBtn.classList.add("incorrect");
+    }
+    Array.from(answerButtons.children).forEach(button => {
+        if(button.dataset.correct === "true"){
+            button.classList.add("correct");
+        }
+        button.disabled = true;
+    });
```

Comment:
94->99 another function :). Name it `showCorrectAnswer();` or alike.

Reply:
?